### ISSUE TITLE
[Bug] aime24 on DeepSeek-R1-Distill-Qwen-1.5B accuracy check

### ISSUE THREAD

**Issue by pathfinder-pf** [sgl-project/sglang-jax#350]
### Checklist

- [x] 1. I have searched related issues but cannot get the expected help.
- [x] 2. The bug has not been fixed in the latest version.
- [x] 3. Please note that if the bug-related issue you submitted lacks corresponding environment info and a minimal reproducible demo, it will be challenging for us to reproduce and resolve the issue, reducing the likelihood of receiving feedback.
- [x] 4. If the issue you raised is not a bug but a question, please raise a discussion at https://github.com/sgl-project/sgl-jax/discussions/new/choose Otherwise, it will be closed.
- [x] 5. Please use English, otherwise it will be closed.

### Describe the bug

the aime24 benchmark on DeepSeek-R1-Distill-Qwen-1.5B can't reproduce 

### Reproduction

-  server
rm -rf /tmp/jit_cache
JAX_COMPILATION_CACHE_DIR=/tmp/jit_cache JAX_USE_SHARDY_PARTITIONER=0 python3 -u -m sgl_jax.launch_server \
--model-path deepseek-ai/DeepSeek-R1-Distill-Qwen-1.5B \
--trust-remote-code  \
--tp-size=1 \
--device=tpu \
--mem-fraction-static=0.8 \
--chunked-prefill-size=2048 \
--download-dir=/tmp \
--dtype=bfloat16 \
--max-running-requests 64 \
--skip-server-warmup \
--page-size=64  \
--max-total-tokens=257536 \
--random-seed=27 \
--precompile-token-paddings=8192 \
--precompile-bs-paddings=64 \
--disable-overlap-schedule 

### Environment

v6e-1
